arrow/request.go

```diff
@@ -0,0 +1,92 @@
+package arrow
+
+import (
+	"context"
+	"io"
+
+	"github.com/tarantool/go-iproto"
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+// INSERT Arrow request.
+//
+// FIXME: replace with iproto.IPROTO_INSERT_ARROW when iproto will released.
+// https://github.com/tarantool/go-tarantool/issues/412
+const iprotoInsertArrowType = iproto.Type(17)
+
+// The data in Arrow format.
+//
+// FIXME: replace with iproto.IPROTO_ARROW when iproto will released.
+// https://github.com/tarantool/go-tarantool/issues/412
+const iprotoArrowKey = iproto.Key(0x36)
+
+// InsertRequest helps you to create an insert request object for execution
+// by a Connection.
+type InsertRequest struct {
+	arrow Arrow
+	space interface{}
+	ctx   context.Context
+}
+
+// NewInsertRequest returns a new empty InsertRequest.
+func NewInsertRequest(space interface{}) *InsertRequest {
```

Comment:
I think this will make using the request a little more useful. But we need to have the `Arrow` method as well.
```suggestion
// NewInsertRequest returns a new empty InsertRequest.
func NewInsertRequest(space interface{}, arrow Arrow) *InsertRequest {
```

Reply:
Yes, such solution was implemented in first PR edition.
But it's not friendly with tests, and this breaks a pattern that is already in use in other similar methods.
If you insist on that solution, I can put it back the way it was before.

Reply:
> But it's not friendly with tests

Why?

> breaks a pattern that is already in use in other similar methods

I think we can do it if it is really more useful. Maybe the old approach is less convenient.

So I prefer the solution `NewInsertRequest(space, arrow)`.

Reply:
> Why?

A lot of cases where we should use empty `Arrow{}`

Any way, I agree with you that suggested signature a bit better.

Reply:
Done.